src/e2e-test/features/bigquerymultitable/sink/BigQueryToBQMT.feature

```diff
@@ -0,0 +1,44 @@
+@BQMT_Sink
+Feature: BQMT-Sink - Verify BQMT sink plugin data transfer scenarios
```

Comment:
BQMT-Sink - Verify BigQuery to BQMT sink plugin data transfer scenarios

Reply:
Change done.